doc/monitoring-stack-deployment/README.md

```diff
@@ -1,3 +1,11 @@
+## Enabling 3scale monitoring stack
+
+This guide goes through the process of enabling 3scale monitoring stack using Prometheus and Grafana Operators.
+Due to Grafana v4 deprecation on OpenShift 4.16+ this guide is divided into two sections, one covering OpenShift versions up to (including) 4.15, second section is specific to OpenShift 4.14+ and 3scale Operator 2.15+.
+It is highly recommended to use Grafana v5 when possible, the guide also goes through the migration steps required to migrate from Grafana v4 to Grafana v5.
+
+#### Common requirement:
+
 1. Enable monitoring in the APIManager CR
```

Comment:
@MStokluska , grafana Dashboards are missing in the initial installation.
To fix the issue, we need to change the order of operations: enabling monitoring in Apimanager must be done after adding Grafana.
Otherwise, we will have the following error and dashboards are missing in Grafana.
Events:
  Type     Reason          Age                From        Message
  ----     ------          ----               ----        -------
  Warning  ReconcileError  79m (x4 over 79m)  APIManager  Error creating grafana dashboard object 'backend'. Install grafana-operator in your cluster to create grafana dashboard objects
  Warning  ReconcileError  79m (x4 over 79m)  APIManager  Error creating grafana dashboard object 'system'. Install grafana-operator in your cluster to create grafana dashboard objects
  Warning  ReconcileError  79m (x4 over 79m)  APIManager  Error creating grafana dashboard object 'zync'. Install grafana-operator in your cluster to create grafana dashboard objects
  Warning  ReconcileError  79m (x4 over 79m)  APIManager  Error creating grafana dashboard object 'apicast-mainapp'. Install grafana-operator in your cluster to create grafana dashboard objects
  Warning  ReconcileError  79m (x3 over 79m)  APIManager  Error creating grafana dashboard object 'apicast-services'. Install grafana-operator in your cluster to create grafana dashboard objects
  Warning  ReconcileError  79m (x3 over 79m)  APIManager  Error creating grafana dashboard object 'kubernetes-resources-by-namespace'. Install grafana-operator in your cluster to create grafana dashboard objects
  Warning  ReconcileError  79m (x3 over 79m)  APIManager  Error creating grafana dashboard object 'kubernetes-resources-by-pod'. Install grafana-operator in your cluster to create grafana dashboard objects

Reply:
Thanks, yes I am aware of it. The other solution is to restart the operator. Will shift the order a bit but in general, the logic around reconciliation triggers did not change with this PR and this is how it was, I was hoping to get an improvement jira for this